app/components/containers/containersController.js

```diff
@@ -239,17 +241,23 @@ angular.module('containers', [])
         return;
       }
     });
+    var title = 'You are about to remove one or more container.';
     if (isOneContainerRunning) {
-      ModalService.confirmDeletion(
-        'You are about to remove one or more running containers.',
-        function (confirmed) {
-          if(!confirmed) { return; }
-          $scope.removeAction();
-        }
-      );
-    } else {
-      $scope.removeAction();
+      title = 'You are about to remove one or more running containers.';
     }
+    var text = 'Automatically clean associated volumes<i></i>';
+    ModalService.confirmContainerDeletion(
+      title,
+      text,
+      function (result) {
+        if(!result) { return; }
+        $scope.cleanAssociatedVolumes = false;
```

Comment:
Instead of using `cleanAssociatedVolumes`, we can initialize an integer with 0 value and use the value of the result (1 if checked) directly. Without having to use that workaround: `v: ($scope.cleanAssociatedVolumes) ? 1 : 0`